src/helpers.js

```diff
@@ -11,106 +11,159 @@ const moment = require('moment');
 .##.....##.########.########.##........########.##.....##..######.
 */
 
-const isTimestamp = label => !!(label && moment.unix(label).isValid());
+const getTimeStamp = (ts) => R.split("_", ts.toString())[0];
```

Comment:
use compose and get the first value

Reply:
Also, I don't see any unit tests covering this

Reply:
Done